### DISCUSSION TITLE
NixOS Search

### DISCUSSION THREAD

**Discussion by sydrinea** [catppuccin/catppuccin#1905] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the requested port be themed?

With [Stylus](https://github.com/openstyles/stylus).

### Has the requested port been themed already?

I've made a draft repo at [https://github.com/alaidriel/ctp-nixos-search](https://github.com/alaidriel/ctp-nixos-search).

**Comment by sydrinea** [comment]
This is ready for review.

**Reply by sgoudham** [reply]
Cool! Transferred over to #1910
